cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
It's not clear whether this config file is strictly necessary, but I've left it in for now.